README.md

```diff
@@ -43,6 +43,30 @@ learn from other people doing development.
 
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
 
+### Running Mailspring from Source
+
+To run Mailspring from its source code, run the following command from the root
+directory of the Mailspring repository:
+
+```
+npm start
```

Comment:
One needs to run `npm install` first, at least the first time (and after each git pull).

Reply:
Added this.